docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -0,0 +1,221 @@
+5. If your project used either the **functions** or **commands** chunks, add them to the chunks list as shown next (the following code is for if your project used both chunks).
+
+    ```javascript
+      new HtmlWebpackPlugin({
+        filename: "taskpane.html",
+        template: "./src/taskpane/taskpane.html",
+        chunks: ["polyfill", "taskpane", "commands", "functions"]
+      })
+    ```
+
+6. Save your changes and rebuild the project.
+
+   ```command line
+   npm run build
+   ```
+
+> [!NOTE]
+> If your project has a **functions.html** file or **commands.html** file, they can be removed. The **taskpane.html** will load the **functions.js** and **commands.js** code into the shared JavaScript runtime via the webpack updates you just made.
+
+## Test your Office Add-in changes
+
+You can confirm that you are using the shared JavaScript runtime correctly by using the following instructions.
+
+1. Open the **manifest.xml** file.
+2. Find the `<Control xsi:type="Button" id="TaskpaneButton">` section and change the following `<Action ...>` XML.
+    
+    from:
+    
+    ```xml
+    <Action xsi:type="ShowTaskpane">
+      <TaskpaneId>ButtonId1</TaskpaneId>
+      <SourceLocation resid="Taskpane.Url"/>
+    </Action>
+    ```
+    
+    to:
+    
+    ```xml
+    <Action xsi:type="ExecuteFunction">
+    <FunctionName>action</FunctionName>
+    </Action>
+    ```
+3. Open the **./src/commands/commands.js** file.
+4. Replace the **action** function with the code below. This will update the function to open and modify the task pane button to increment a counter. Opening and accessing the task pane DOM from a command only works with the shared JavaScript runtime.
+    
+    ```javascript
+    var _count=0;
+    
+    function action(event) {
+      // Your code goes here
+      _count++;
+      Office.addin.showAsTaskpane();
+      document.getElementById("run").textContent="Go"+_count;
+    
+      // Be sure to indicate when the add-in command function is complete
+      event.completed();
+    }
+    ```
+
+5. Save your changes and run the project.
+
+   ```command line
+   npm start
+   ```
+
+Each time you select the add-ins button, it will change the **run** button text to **go** and increment a counter after it.
+
+## Runtime lifetime
+
+When you add the `Runtime` element, you also specify a lifetime with a value of `long` or `short`. Set this value to `long` to take advantage of features such as starting your add-in when the document opens, continuing to run code after the task pane is closed, or using CORS and DOM from custom functions.
+
+>[!NOTE]
+> The default lifetime value is `short`, but we recommend using `long` in Excel add-ins. If you set your runtime to `short` in this example, your Excel add-in will start when one of your ribbon buttons is pressed, but it may shut down after your ribbon handler is done running. Similarly your add-in will start when the task pane is opened, but it may shut down when the task pane is closed.
```

Comment:
```suggestion
> The default lifetime value is `short`, but we recommend using `long` in Excel add-ins. If you set your runtime to `short` in this example, your Excel add-in will start when one of your ribbon buttons is pressed, but it may shut down after your ribbon handler is done running. Similarly, your add-in will start when the task pane is opened, but it may shut down when the task pane is closed.
```